dclab/definitions/meta_const.py

```diff
@@ -98,6 +98,14 @@
         ["roi size x", fint, "Image width [px]"],
         ["roi size y", fint, "Image height [px]"],
     ],
+    # All qpi-related keywords
+    "qpi": [
```

Comment:
Does this information belong here? How descriptive should we be here?

If it does belong, it means the user should fill this info in when writing qpi data to an RTDC file, right?

Reply:
Yes, I think it belongs here. A few comments:

- please put "qpi" further below (alphabetical order)
- I would replace "laser lambda" by just "wavelength". I know this would be different than in the "fluorescence" section, but this also helps to distinguish the two sections and this is also how we do it in qpimage etc. Furthermore, it does not necessarily have to be a laser ;)
- I would also use "index medium" which is shorter and aligns with the qpimage notation: https://github.com/RI-imaging/qpimage/blob/master/qpimage/meta.py
- I believe the "alpha" values are too specific. They would probably fit better in a "calculation" section and should be discussed at a later time point (Looks like an algorithm is needed that identifies RBCs first, i.e. an `ml_score_rbc` or so might be needed. Does it make sense to have a `qpi_refinc` (refraction increment) feature?

The medium index is something that the acquisition software can set automatically when the user selects a medium. Alternatively, dclab could also infer it from the `[setup]:medium` key, given that we have consistent RI values for the standard RT-DC buffers (which I am not aware of).

Reply:
Maybe it also makes sense to add the information about the hologram analysis (sideband position, fourier filter name and size, padding, software used (matlab script name, qpretrieve with version), etc.) to make the analysis pipeline reproducible.

Reply:
Because Refractive increment is cell-specific, I think it's too changeable to be a feature. For example, if you are calculating the dry mass, the `qpi_refinc` would change depending on the classification of the cell, right? We could have a dictionary feature for the cell type, but dicts might be a bit awkward to deal with?

```python
qpi_refinc = {
    'ml_score_rbc`: 0.04
    'ml_score_bas`: 0.095
    ...
}
```

I would think that these values are configuration values that are rather consistent for each cell type, and could be updated by the user if a new one is needed. 

So the calculation section you mention would probably be best! I don't know if you have implemented something like that  before though!

> The medium index is something that the acquisition software can set automatically when the user selects a medium. Alternatively, dclab could also infer it from the [setup]:medium key, given that we have consistent RI values for the standard RT-DC buffers (which I am not aware of).

Nice idea. But we should leave our qpi section's "medium index" there though, right?

> Maybe it also makes sense to add the information about the hologram analysis (sideband position, fourier filter name and size, padding, software used (matlab script name, qpretrieve with version), etc.) to make the analysis pipeline reproducible.

I like it, will do!